app/controllers/clearance/passwords_controller.rb

```diff
@@ -85,8 +86,10 @@ def find_user_by_id_and_confirmation_token
   end
 
   def find_user_for_create
+    password_params = params[:password] || {}
```

Comment:
The params are different in each place (`params[:password]` and `params[:password_reset]`). I thought about extracting the methods anyway (extracting two of them), as I don't usually like local variables that are easily avoided, but every method I add to a controller essentially becomes "public api".